test/integration/lib/confirm-sig-requests.js

```diff
@@ -17,7 +17,7 @@ QUnit.test('successful confirmation of sig requests', (assert) => {
 
 global.ethQuery = global.ethQuery || {}
 
-async function runConfirmSigRequestsTest (assert, done) {
```

Comment:
lol how?

Reply:
Hmm? It was unused? It also wasn't passed in above:

```js
runConfirmSigRequestsTest(assert).then(done).catch((err) => {
  assert.notOk(err, `Error was thrown: ${err.stack}`)
  done()
})
```